End2EndTests/ArmoniK.EndToEndTests.Client/Tests/PriorityClient.cs

```diff
@@ -52,15 +52,15 @@ public async Task Priority()
       var taskDefinitions = new List<TaskDefinition>();
       for (var i = 0; i < nTasksPerSessionPerPriority; i++)
       {
-        var priorityBlobInfo = await Client.BlobService.CreateBlobAsync(SessionHandle,
-                                                                        "Priority",
-                                                                        Encoding.UTF8.GetBytes(priority.ToString()))
-                                           .ConfigureAwait(false);
+        var priorityBlobInfo = await Client!.BlobService.CreateBlobAsync(SessionHandle!,
+                                                                         "Priority",
+                                                                         Encoding.UTF8.GetBytes(priority.ToString()))
+                                            .ConfigureAwait(false);
         var priorityBlobHandle = new BlobHandle(priorityBlobInfo,
                                                 Client);
 
         var resultName = "Result" + priority;
-        var taskDefinition = new TaskDefinition().WithLibrary(WorkerLibrary)
+        var taskDefinition = new TaskDefinition().WithLibrary(WorkerLibrary!)
```

Comment:
WorkerLibrary should not be nullable. Either it exists or we cannot create it and it's an error